nodes/src/nodes/multi_agent/blackboard.py

```
@@ -0,0 +1,105 @@
+                    agent_name=agent_name,
+                    key=key,
+                    value=copy.deepcopy(value),
+                    timestamp=time.time(),
+                )
+            )
+
+    # ------------------------------------------------------------------
+    # Read
+    # ------------------------------------------------------------------
+
+    def read(self, key: str) -> Optional[Any]:
```

Ruff (0.15.7)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `read`

(ANN401)